### PR TITLE
SCP-2149: Better GH issue templates

### PR DESCRIPTION
**Pull request by j-mueller**
Pre-submit checklist:
- Branch
    - [ ] Commit sequence broadly makes sense
    - [ ] Key commits have useful messages
    - [ ] Relevant tickets are mentioned in commit messages
    - [ ] Formatting, materialized Nix files, PNG optimization, etc. are updated
- PR
    - [ ] Self-reviewed the diff
    - [ ] Useful pull request description
    - [ ] Reviewer requested

Pre-merge checklist:
- [ ] Someone approved it
- [ ] Commits have useful messages
- [ ] Review clarifications made it into the code
- [ ] History is moderately tidy; or going to squash-merge

### CODE REVIEW

**Comment by j-mueller** [comment]
Not sure if we need the performance one.

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 1]
```diff
@@ -0,0 +1,43 @@
+---
+name: Bug report
+about: Report a bug
+title: "[BUG] - "
+labels: bug
+assignees: ''
+
+---
+
+**Internal/External**
```
Comment (michaelpj) [inline .github/ISSUE_TEMPLATE/bug_report.md:10]: It wasn't immediately obvious to me what to do with these - can we make them checkboxes insetead?

Comment (michaelpj) [reply .github/ISSUE_TEMPLATE/bug_report.md:10]: Do we need this one in particular? I think it'll likely be obvious from the username. Maybe it's still helpful.

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+*Internal*   if an IOHK staff member.
+*External*   otherwise.
+
+**Area**
+*PlutusTx*    Related to the GHC plugin, Haskell-to-Plutus compiler, on-code
```
Comment (michaelpj) [inline .github/ISSUE_TEMPLATE/bug_report.md:15]:
```suggestion
*PlutusTx*    Related to the GHC plugin, Haskell-to-Plutus compiler, on-chain code
```

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+*PAB*    Related to the Plutus application backend (PAB), emulator, Plutus libraries
+*Marlowe* Related to Marlowe
+*Other*   Any other topic (Playgrounds, etc.)
+
+**Summary**
```
Comment (michaelpj) [inline .github/ISSUE_TEMPLATE/bug_report.md:20]: Why not headings? i.e. 
```
## Summary`
```
or whatever

File: .github/ISSUE_TEMPLATE/feature_request.md [hunk 4]
```diff
@@ -0,0 +1,27 @@
+---
+name: Feature request
+about: Submit a feature request
+title: "[FR] - "
```
Comment (michaelpj) [inline .github/ISSUE_TEMPLATE/feature_request.md:4]: Do we need the title prefixes if we have the labels?

File: .github/ISSUE_TEMPLATE/performance_issue.md [hunk 5]
```diff
@@ -0,0 +1,35 @@
+---
```
Comment (michaelpj) [inline .github/ISSUE_TEMPLATE/performance_issue.md:1]: Are we actually interested in these right now?

Comment (michaelpj) [reply .github/ISSUE_TEMPLATE/performance_issue.md:1]: And is a noticeable performance issue different from a bug?

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 6]
```diff
@@ -0,0 +1,43 @@
...
+
+**Internal/External**
+*Internal*   if an IOHK staff member.
+*External*   otherwise.
+
+**Area**
+*PlutusTx*    Related to the GHC plugin, Haskell-to-Plutus compiler, on-code
+*PAB*    Related to the Plutus application backend (PAB), emulator, Plutus libraries
+*Marlowe* Related to Marlowe
+*Other*   Any other topic (Playgrounds, etc.)
```
Comment (michaelpj) [inline .github/ISSUE_TEMPLATE/bug_report.md:18]: It's a shame you can't do conditional tagging based on checkboxes or something...

File: .github/ISSUE_TEMPLATE/feature_request.md [hunk 7]
```diff
@@ -0,0 +1,27 @@
...
+assignees: ''
+
+---
+
+**Internal/External**
+*Internal*   if an IOHK staff member.
+*External*   otherwise.
+
+**Area**
+*PlutusTx*    Related to the GHC plugin, Haskell-to-Plutus compiler, on-code
```
Comment (michaelpj) [inline .github/ISSUE_TEMPLATE/feature_request.md:15]:
```suggestion
*PlutusTx*    Related to the GHC plugin, Haskell-to-Plutus compiler, on-chain code
```

File: .github/ISSUE_TEMPLATE/bug_report.md (hunk 6 above)
Comment (michaelpj) [reply .github/ISSUE_TEMPLATE/bug_report.md:18]: Also I wonder if we should use one of our more "product"-y breakdowns, e.g. "PlutusTx -> Plutus Foundation", "PAB -> Plutus Application Framework"?

File: .github/ISSUE_TEMPLATE/bug_report.md (hunk 1 above)
Comment (j-mueller) [reply .github/ISSUE_TEMPLATE/bug_report.md]: I copied that from the node repository. But I agree we probably don't need it, this information is already on Github.

File: .github/ISSUE_TEMPLATE/bug_report.md (hunk 6 above)
Comment (j-mueller) [reply .github/ISSUE_TEMPLATE/bug_report.md]: Yeah good idea.

File: .github/ISSUE_TEMPLATE/bug_report.md (hunk 3 above)
Comment (j-mueller) [reply .github/ISSUE_TEMPLATE/bug_report.md]: Fixed.

File: .github/ISSUE_TEMPLATE/feature_request.md (hunk 4 above)
Comment (j-mueller) [reply .github/ISSUE_TEMPLATE/feature_request.md]: No, I suppose not

File: .github/ISSUE_TEMPLATE/bug_report.md (hunk 1 above)
Comment (michaelpj) [reply .github/ISSUE_TEMPLATE/bug_report.md]: Yeah, and with these things the longer we make it the more likely people just ignore it.

**Review by michaelpj — APPROVED** [review]

**Pull request merged**